src/onnx/onnx.cpp

```diff
@@ -427,8 +444,8 @@ struct onnx_parser
     instruction_ref process_auto_pad_attribute(instruction_ref ins,
                                                node_info info,
                                                Op& op,
-                                               std::array<std::size_t, 2> k_lens,
-                                               std::array<std::size_t, 2> dilation,
+                                               std::vector<std::size_t> k_lens,
+                                               std::vector<std::size_t> dilation,
```

Comment:
Again, we need to check the dimensions of k_lens, dilation, in_lens match each other.